R/icr.R

```diff
@@ -89,7 +89,7 @@ test_icr <- function(data, unit_var, coder_var, ...,
     })
     # Bind all data frames together and reorder resulting data frame
     out <- dplyr::bind_rows(out) %>%
-      dplyr::select(group, tidyselect::everything())
+      dplyr::select(.data$group, tidyselect::everything())

```

Comment:
When testing the package, this change raises a warning:
```
Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"group"` instead of `.data$group`
```